modules/skeletonRetriever/src/main.cpp

```diff
@@ -276,6 +276,7 @@ class Retriever : public RFModule
             if (irgbd->getRgbFOV(fov_h, fov_v)) {
                 yInfo() << "camera fov_h (from sensor) =" << fov_h;
                 yInfo() << "camera fov_v (from sensor) =" << fov_v;
+                rgbdDrv.close();
```

Comment:
`getCameraOptions` shouldn't have the responsibility to close the driver.
A better place where to do that is right where the function gets called.

I'll modify this after the merge.